databricks/koalas/frame.py

```diff
@@ -2350,6 +2350,134 @@ def fillna(self, value=None, axis=None, inplace=False):
+            The value to be replaced must be an int, float, or string.
+        value : int, float, string, or list
+            Value to use to replace holes. The replacement value must be an int, float,
+            or string. If value is a list, value should be of the same length with to_replace.
+        subset : sting, list
+            Optional list of column names to consider. Columns specified in subset that
+            do not have matching data type are ignored. For example, if value is a string,
+            and subset contains a non-string column, then the non-string column is simply ignored.
+        inplace : boolean, default False
+            Fill in place (do not create a new object)
+
+        Returns
+        -------
+        DataFrame
+            Object after replacement.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({"name": ['Ironman', 'Captain America', 'Thor', 'Hulk'],
+        ...                    "weapon": ['Mark-45', 'Shield', 'Mjolnir', 'Smash']})
+        >>> df
+                      name   weapon
+        0          Ironman  Mark-45
+        1  Captain America   Shield
+        2             Thor  Mjolnir
+        3             Hulk    Smash
+
+        Scalar `to_replace` and `value`
+
+        >>> df.replace('Ironman', 'War-Machine')
+                      name   weapon
+        0      War-Machine  Mark-45
+        1  Captain America   Shield
+        2             Thor  Mjolnir
+        3             Hulk    Smash
+
+        List like `to_replace` and `value`
+
+        >>> df.replace(['Ironman', 'Captain America'], ['Rescue', 'Hawkeye'], inplace=True)
+        >>> df
+              name   weapon
+        0   Rescue  Mark-45
+        1  Hawkeye   Shield
+        2     Thor  Mjolnir
+        3     Hulk    Smash
+
+        Replacing value by specifying column
+
+        >>> df.replace('Mjolnir', 'Stormbuster', subset='weapon')
+              name       weapon
+        0   Rescue      Mark-45
+        1  Hawkeye       Shield
+        2     Thor  Stormbuster
+        3     Hulk        Smash
+
+        Dict like `to_replace`
+
+        >>> df = ks.DataFrame({'A': [0, 1, 2, 3, 4],
+        ...                    'B': [5, 6, 7, 8, 9],
+        ...                    'C': ['a', 'b', 'c', 'd', 'e']})
+
+        >>> df.replace({'A': {0: 100, 4: 400}})
+             A  B  C
+        0  100  5  a
+        1    1  6  b
+        2    2  7  c
+        3    3  8  d
+        4  400  9  e
+
+        >>> df.replace({'A': 0, 'B': 5}, 100)
+             A    B  C
+        0  100  100  a
+        1    1    6  b
+        2    2    7  c
+        3    3    8  d
+        4    4    9  e
+
+        Notes
+        -----
+        One difference between this implementation and pandas is that it is necessary
+        to specify the column name when you are passing dictionary in `to_replace`
+        parameter. Calling `replace` like `df.replace({0: 10, 1: 100})` will
```

Comment:
Maybe it's better to say... 

``Calling `replace` on its index such as `df.replace ...` ``